blt-project/blt/sites/example.yml

```diff
@@ -0,0 +1,21 @@
+# @file
+# example.yml
+# Site definition file for remote and local environment information.
+#
+# The filename of this file should correspond to docroot/sites/[site-name]
+# (e.g. default.yml for docroot/sites/default). You should create one file in
+# this directory for each site running against your codebase.
+aliases:
```

Comment:
Can we continue to use drush.aliases here rather than introducing something new?